### PR TITLE
Added docs for two Kafka Connect MongoDB sink

### LINKED ISSUE #845: Document MongoDB Sink Connector
# What should we add to our content collections?

(more information is useful here! Were you trying to do something particular?)

Document MongoDB Sink Connector

# Is this something you would work on yourself?

[X] I plan to open a pull request for this

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Added docs for:
* Standard MongoDB sink connector by MongoDB
* Stream reactor MongoDB sink connector by Lenses.io

Fixes #845 #846

### CODE REVIEW

**Comment by lornajane** [comment]
Do you have a preferred mongo cloud provider that you tested this with? I'm not sure if your reviewer should use the same or intentionally choose a different one!

**Comment by ftisiot** [comment]
I created a VM on GCP and tested it there, didn't check for cloud providers but indeed testing with a different one might be a good idea

File: docs/products/kafka/kafka-connect/howto/mongodb-sink-mongo.rst [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+Define the connector configurations in a file (we'll refer to it with the name ``mongodb_sink.json``) with the following content, creating a file is not strictly necessary but allows to have all the information in one place before copy/pasting them in the `Aiven Console <https://console.aiven.io/>`_:
+
+.. code-block:: json
+
+    {
+        "name":"CONNECTOR_NAME",
+        "connector.class": "com.mongodb.kafka.connect.MongoSinkConnector",
+        "topics": "TOPIC_LIST",
+        "connection.uri": "MONGODB_CONNECTION_URI",
```
Comment (anelook) [inline docs/products/kafka/kafka-connect/howto/mongodb-sink-mongo.rst:63]: Similar point on the connection.uri as in the other article! the definition below is way more helpful :)

**Review by anelook — APPROVED** [review]
Looked through the article, it uses the same building blocks as other connectors. Created 
 test connectors for both of the sink types.

I have only couple of minor comments.

**Pull request merged**